samples/metaapp/src/main/java/com/sourcepointmeta/metaapp/tv/properties/PropertyCardView.kt

```diff
@@ -23,6 +23,7 @@ fun PropertyCardView.bind(item: PropertyDTO) {
     account_id.text = "${item.accountId}"
     gdpr_icon.setImageResource(getResStatusResIcon(item.gdprEnabled))
     ccpa_icon.setImageResource(getResStatusResIcon(item.ccpaEnabled))
+    ccpa_icon.setImageResource(getResStatusResIcon(item.usnatEnabled))
```

Comment:
I was also talking about `ccpa_icon.setImageResource`, I think it should be something like `usnat_icon.setImageResource`, or am I wrong?

Reply:
I think you are right

Reply:
but this is the tv section, so we'll leave for later